css/src/atomics/colors.scss

```
@@ -69,6 +48,54 @@
 	}
 }
 
+@include tablet {
```

This is a minor change, but I think that we should keep control flow on the outermost indent whenever possible. Media queries can follow inside. Thoughts? I think it would be good to standardize this, since we'll be writing loops in other places too.

```suggestion
@each [...] {
   $vars: value;
   
   @tablet {
   
   }
}


```